testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -471,7 +478,7 @@ impl ClusterSwarmKube {
                 self.fullnode_spec(
                     fullnode_config.fullnode_index,
                     fullnode_config.num_fullnodes_per_validator,
-                    instance_config.validator_group,
+                    instance_config.validator_group.index as u32,
```

Comment:
Problem with this part is that currently twin index can be set anywhere, but if node is not validator it is simply ignored. Ideally we want to fail(panic) in this case to let programmer know that they used twin index on instance type that does not support it

Can you replace this with `.index_only()->u32` method, and in this method panic if twin index is set.
Use it here and below in all places where twin index is not supported.